using_images/db_images/mongodb.adoc

```diff
@@ -266,48 +122,9 @@ MongoDB settings can be configured with the following environment variables.
 |true
 |===
 
-*Volume Mount Points*
-
-You can set volume mount points with the following command:
-
-****
-`$ docker run -v /_<host>_:/_<container>_`
-****
-
-.Volumes
-[cols="3a,3a",options="header"]
-|===
-
-|Volume mount point |Description
-
-|`/var/lib/mongodb/`
-|MongoDB data directory.
-|===
-
-=== MongoDB Root User
-The root user is not set by default. You can create one when initializing the
-database by setting the `MONGODB_ROOT_PASSWORD` environment variable. If you set
-this environment variable, then the root user name is set to `admin`.
+=== Volume Mount Points
 
-== Image Name Structure
+The MongoDB image can be be run with mounted volumes to enable persisted storage for the database:
```

Comment:
technically i think either is correct, but persistent is more standard, i'll change it.